buddyboss/buddyboss-platform
# PROD-805 Recently Active Members widget "More >" button issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2484 .

### How to test the changes in this Pull Request:

1. Add Recently Active Members widget somewhere.
2. Set Max members to show some higher value then your actual active members
3. You should not see the ( More > ) button there.

### Proof Screenshots or Video

https://www.loom.com/share/013fd0b597c54c0d8c43a4abc04e8bb6

### Other information:




### Changelog entry

> Recently Active Members widget more button issue fixed
